mava/components/jax/updating/parameter_server.py

```diff
@@ -204,3 +204,58 @@ def on_parameter_server_run_loop(self, server: SystemParameterServer) -> None:
             server.store.system_checkpointer.save()
             server.store.last_checkpoint_time = time.time()
             print("Updated variables checkpoint.")
+
+
+class ParameterServerSeparateNetworks(DefaultParameterServer):
```

Comment:
Same with the ParameterServer. What is the reason we can't build a generic one that works with both network setups? Is it that there would be too many if statements?

Reply:
I don't see there being any issues with doing this. The only reason I did it was to avoid if statements (and potential bugs) but very happy to change it so that we only have generic ones for both.

Reply:
I was wondering if it would be possible without it statements. I agree that if statements is what we are trying to avoid :)

Reply:
I couldn't find a nice way of doing this without adding in an extra if statement. But happy to make changes if you can spot a way :)

Reply:
It think it is fine for now :)